lib/vtls/schannel.c

```diff
@@ -337,6 +339,17 @@ set_ssl_ciphers(SCHANNEL_CRED *schannel_cred, char *ciphers)
     long alg = strtol(startCur, 0, 0);
     if(!alg)
       alg = get_alg_id_by_name(startCur);
+    if(alg == -1) {
```

Comment:
shouldn't this rather be `if(alg == eNULL)` for better readability ?

Reply:
good point. i'll do the change as soon, as I can.